modules/local_storage/src/Plugin/QueueWorker/StockLevelUpdater.php

```diff
@@ -22,10 +22,10 @@ class StockLevelUpdater extends QueueWorkerBase {
    */
   public function processItem($data) {
     $entity_id = $data;
-    $updater = \Drupal::service('commerce_stock.local_stock_service')->getStockUpdater();
-    $locations = $updater->getLocationList(TRUE);
+    $checker = \Drupal::service('commerce_stock.local_stock_service')->getStockChecker();
```

Comment:
getLocationList() is on the checker and updateLocationStockLevel() needs to move to the updater. As a result this function should use both.
I think we may handle this in https://github.com/BBGuy/commerce_stock/pull/19.